stubs/JACK-Client/@tests/stubtest_allowlist.txt

```diff
@@ -0,0 +1,2 @@
+# The available constants differ based on the local environment.
+jack\.METADATA_.*
```

Comment:
If your local install now sees all the `METADATA_ *` constants. Wouldn't this now fail for you locally with an "unused allowlist entry" error?

I guess the idea is that it's fine to fail for you, as long as it passes on the CI.

Still I'm really curious about why it's different. My local environment aggrees with the CI
My setup:
![image](https://github.com/user-attachments/assets/1d9416bf-9a37-40a2-bb3d-b099605c8164)
![image](https://github.com/user-attachments/assets/0e7a211b-d466-4fc4-bf6a-985a00827f25)
![image](https://github.com/user-attachments/assets/283c3a73-4d2c-498f-86a0-91a4c238cb00)

Reply:
> Wouldn't this now fail for you locally with an "unused allowlist entry" error?

AFAIK there are some heuristics that prevent that message for (some) regexps.

Reply:
I'm using Ubuntu 24.10, which has a slightly newer libjack-dev version (1:0.126.0-2build3). I'd assume that that's the reason the constants are now present. But testing it again, I now get

```
note: unused allowlist entry jack\.METADATA_.*
```

I'll fix the allowlist.